build/vsts-ci.yml

```diff
@@ -189,6 +189,7 @@ phases:
   variables:
     BuildConfig: Release
     OfficialBuildId: $(BUILD.BUILDNUMBER)
+    DotnetVersionKind: $[variables.VERSIONKIND] # If not "VERSIONKIND" variable is set when queuing the publishing task, this defaults to empty string.
```

Comment:
nit: If no "VERSIONKIND" variable is set...

Reply:
Resolved.